lib/pages/admin.dart

```diff
@@ -0,0 +1,152 @@
+        if (users.hasData) {
+          return Padding(
+            padding: const EdgeInsets.only(top: 10),
+            child: ListView.builder(
+              shrinkWrap: true,
+              physics: const NeverScrollableScrollPhysics(),
+              controller: _scrollController,
+              itemCount: users.data?.length,
+              itemBuilder: (BuildContext context, int index) {
+                return Column(
+                  crossAxisAlignment: CrossAxisAlignment.center,
+                  mainAxisSize: MainAxisSize.min,
+                  children: <Widget>[
+                    Card(
+                      clipBehavior: Clip.antiAlias,
+                      margin: const EdgeInsets.only(
+                          bottom: 10, left: 10, right: 10),
+                      child: Theme(
+                        data: Theme.of(context)
+                            .copyWith(dividerColor: Colors.transparent),
+                        child: ExpansionTile(
+                          key: GlobalKey(),
+                          initiallyExpanded: false,
+                          title: Text(
+                            "${users.data?[index]['first_name']} ${users.data?[index]['last_name']}",
+                            style: const TextStyle(
+                                fontSize: 20.0, fontWeight: FontWeight.bold),
+                          ),
+                          trailing: const Icon(Icons.edit),
+                          controlAffinity: ListTileControlAffinity.leading,
+                          children: <Widget>[
+                            Padding(
+                              padding: const EdgeInsets.only(
+                                  bottom: 16, left: 16, right: 16),
+                              child: Align(
+                                alignment: Alignment.topLeft,
+                                child: Column(
+                                  mainAxisSize: MainAxisSize.min,
+                                  children: <Widget>[
+                                    Align(
+                                      alignment: Alignment.centerLeft,
+                                      child: Text(
+                                          "Email: ${users.data?[index]['email']}"),
+                                    ),
+                                    Align(
+                                      alignment: Alignment.centerLeft,
+                                      child: Text(
+                                          "Telefon nummer: ${users.data?[index]['phone_number']}"),
+                                    ),
+                                    Align(
+                                      alignment: Alignment.centerLeft,
+                                      child: Text(
+                                          "Lokation: ${users.data?[index]['location']}"),
+                                    ),
+                                  ],
```

Comment:
This is like the only place we use dansih in the app, so i think to keep the consistency, it should be englisg aswell

Reply:
Changed to english